diff -ruN nonpatched/bin/jdtls.py patched/bin/jdtls.py
--- nonpatched/bin/jdtls.py	2024-05-30 20:15:43.220000000 +0700
+++ patched/bin/jdtls.py	2024-06-29 12:58:28.887905142 +0700
@@ -23,13 +23,7 @@
 	if known_args.java_executable is not None:
 		java_executable = known_args.java_executable
 	else:
-		java_executable = 'java'
-
-		if 'JAVA_HOME' in os.environ:
-			ext = '.exe' if platform.system()  == 'Windows' else ''
-			java_exec_to_test = Path(os.environ['JAVA_HOME']) / 'bin' / f'java{ext}'
-			if java_exec_to_test.is_file():
-				java_executable = java_exec_to_test.resolve()
+		return 'java_executable_to_set'
 
 	if not known_args.validate_java_version:
 		return java_executable
